cmd/minishift/cmd/start.go

```diff
@@ -359,6 +364,16 @@ func autoMountHostFolders(driver drivers.Driver) {
 	}
 }
 
+func addProfileToConfig() {
```

Comment:
Is this handling the case where one tries to start a new instance using an existing profile name? Some test would be nice here.